pulsar-transaction/coordinator/src/main/java/org/apache/pulsar/transaction/coordinator/impl/TxnLogBufferedWriter.java

```diff
@@ -541,8 +539,53 @@ public void recycle(){
             this.asyncAddArgsList.clear();
             this.handle.recycle(this);
         }
+
+        public void addCallback(AddDataCallback callback, Object ctx){
+            AsyncAddArgs asyncAddArgs = AsyncAddArgs.newInstance(callback, ctx, System.currentTimeMillis());
+            asyncAddArgsList.add(asyncAddArgs);
+        }
     }
 
+    /** Callback for batch write BK. **/
+    private final BufferedAddEntryCallback bufferedAddEntryCallback = new BufferedAddEntryCallback();
```

Comment:
Let's place this together with all class variables in the beginning of the class

Again, comments: `/** Callback for batch write BK. **/` then
`bufferedAddEntryCallback` --> `bookKeeperBatchedWriteCallback`
then delete comment

Reply:
Thank you for your example, already fixed